waiter/src/waiter/reporter.clj

```diff
@@ -87,15 +100,55 @@
                s/Any s/Any}
               (merge {:pickled? true} config)))
 
+(defn- make-reporting-name
```

Comment:
We don't seem to be using the varargs version (found only one caller). Can we just replace this with `(str prefix "." k)` in the caller?